src/main/java/lotto/domain/LottoGroup.java

```diff
@@ -0,0 +1,57 @@
+package lotto.domain;
+
+import java.util.*;
+
+public class LottoGroup {
+    private final List<Lotto> lottos;
+
+    private LottoGroup(List<Lotto> lottos) {
+        this.lottos = Collections.unmodifiableList(new ArrayList<>(lottos));
+    }
+
+    public LottoGroup(Lotto... lottos) {
+        this.lottos = Arrays.asList(lottos.clone());
```

Comment:
자기 생성자 호출하는 `this()`를 사용하면 어떨까요?

```suggestion
        this(Arrays.asList(lottos));
```